database.go

```
@@ -332,7 +369,7 @@ func (m *Measurement) idsForExpr(n *influxql.BinaryExpr) (seriesIDs, bool, influ
 	}
 
 	// ignore time literals
-	if _, ok := value.(*influxql.TimeLiteral); ok {
+	if _, ok := value.(*influxql.TimeLiteral); ok || name.Val == "time" {
```

no, that should return a parse error because that's not a valid time.